### ISSUE TITLE
Create methods to view admins, owners and operators

### ISSUE THREAD

**Issue by re1ro** [axelarnetwork/axelar-cgp-solidity#67]
Currently the only way to read admins, owners or operators for given epoch (and their threshold values) is through reading specific eternal storage slots.

We should create methods to easily access that data. There are several ways to do this:
1. Adding according methods to the Gateway itself (this will increate the contract size that is already close to the limit)
1. Adding the methods to our JavaScript library so the according storage slots can be computed locally and then read from the contract
1. Accessing that data from the Gateway event history. For each operator/ownership transfer we emit events containing required information
```solidity
    event OperatorshipTransferred(
        address[] preOperators,
        uint256 prevThreshold,
        address[] newOperators,
        uint256 newThreshold
    );
```

The gas saving approach would be reading the data from events and JS library and eventually stop saving the list of operators to the gateway storage. Event history can be considered as such storage that is also costly. End simplifying the event structure by dropping `preOperators` could save gas even more as this data can be found in the previous event:
```solidity
    event OperatorshipTransferred(
        uint256 newEpoch,
        address[] newOperators,
        uint256 newThreshold
    );
```

**Comment by milapsheth** [comment]
Owners and operators can be read through external methods currently. But `admins` cannot and all the thresholds cannot. At the least, for transparency we should allow reading `admins` from the contract/explorer itself. Maybe we can save space in another way.

**Comment by milapsheth** [comment]
Fixed #81
